### PR TITLE
Update Travis-CI to utilize new Dockerfile structure

### PR DESCRIPTION
**Pull request by MasterOdin**
Next step of #21 wherein Travis has been updated to utilize the new Dockerfile but dynamically testing on different python versions.

This has been tested at https://travis-ci.org/MasterOdin/asciidoc3/builds/522404414 where I've got print lines for Docker to ensure the proper python version is being tested.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-install:
-  - sudo apt-get update -yq2
-  - sudo apt-get install -yq2 --no-install-recommends dvipng graphviz imagemagick lilypond source-highlight texlive-latex-base
+
+service:
+  - docker
+
+before_script:
+  # Get the MAJOR.MINOR version of Python
+  - export PYTHON_VERSION=$(python -c "from __future__ import print_function; import sys; print('.'.join(map(str,sys.version_info[0:2])))")
```
Comment (cclauss) [inline .travis.yml]: export PYTHON_VERSION=$(python -c "import sys; print('{}.{}'.format(*sys.version_info))”)

Comment (MasterOdin) [reply .travis.yml]: Good point.

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +1,27 @@
 dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
```
Comment (cclauss) [inline .travis.yml]: This line can be deleted because Xenial in now Travis’ default distro.

File: .travis.yml [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
 language: python
+
 python:
   - 3.4
```
Comment (cclauss) [inline .travis.yml]: 3.4 is end of life.

File: .travis.yml (hunk 2 above)
Comment (MasterOdin) [reply .travis.yml]: Done in https://github.com/asciidoc/asciidoc-py3/pull/77

File: .travis.yml (hunk 3 above)
Comment (MasterOdin) [reply .travis.yml]: Done in https://github.com/asciidoc/asciidoc-py3/pull/77

**Comment by MasterOdin** [comment]
Closing this as the only reason to involve docker would have been to validate slidy tests, but it's being ignored for now.

**Pull request closed without merging**